chore: updates kommander on-premise installation tutorial. (#3111)
mesosphere/dcos-docs-site: pages/ksphere/kommander/1.1/tutorials/on-prem/index.md

Once you have both `cluster.yaml` file and `inventory.yaml` file, you need to add a provisioner for your provider. Follow these steps:

Make sure you have selected a workspace - for example the "Default Workspace".

> - Navigate to **Administration / Infrastructure Providers**.

Reviewer: The previous list style (1. 2. 3. 4.) seems better to me in the context of sequential steps and I don't think the empty lines are necessary.

Author: It did not fully work the way it was due to the embedded HTML. Note how the numbers jump around.

<img width="269" alt="Screenshot 2020-07-23 at 14 38 45" src="https://user-images.githubusercontent.com/592445/88287206-46312800-ccf2-11ea-93bc-35c5744f04d6.png">

> - Navigate to **Administration / Infrastructure Providers**.

Reviewer 2: ```suggestion
- Go to **Administration / Infrastructure Providers**.
```

- Select the **Add Infrastructure Provider** button.

- Select **On Premise**.

> - Enter a name for your provider and insert the full contents of your private ssh key, then hit **Verify** and **Save**.

Reviewer 2: ```suggestion
- Enter a name for your provider and insert the full contents of your private ssh key, then select **Verify** and **Save**.
```

Go to the **Clusters** section and select **Add Cluster**.

![Add Cluster](/ksphere/kommander/1.1/img/clusters-header.png)

Select **Upload YAML** to create a cluster.

![Upload YAML](/ksphere/kommander/1.1/img/add-cluster.png)

#### Fill out the form

> - Enter a unique name for your cluster.

Reviewer: Longer steps here thus it makes sense to change them as you did 👍 .

> - At the top of the `cluster.yaml` text field, fill the following information:

Reviewer 2: ```suggestion
- At the top of the `cluster.yaml` text field, enter the following information:
```

---
```

> <p class="message--important"><strong>IMPORTANT: </strong>You must paste the above <i>ClusterProvisioner</i> into the <i>cluster.yaml</i> field and then add the contents of your <code>cluster.yaml</code> after the three dashes. Replace <i>&lt;username&gt;</i> with the SSH user tied to the keys pushed to your machines. This is the same user tied to the key uploaded in the cloud provider step.</p>

Reviewer: The use of italics for `ClusterProvisioner` and `cluster.yaml` seems non-conventional, why not keeping it as it was?

Author: This is HTML - it is broken when leaving as is.

<img width="916" alt="Screenshot 2020-07-23 at 14 37 16" src="https://user-images.githubusercontent.com/592445/88287039-0cf8b800-ccf2-11ea-81e5-597a965a7e21.png">

> - Make sure the Kommander addon is disabled with `enabled: false`.

Reviewer 2: ```suggestion
- Ensure the Kommander addon is disabled with `enabled: false`.
```
